### PR TITLE
yml file updated

### PR DESCRIPTION
**Pull request by anjaliahlawat**

### CODE REVIEW

File: .github/workflows/amazon.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ name: Qa login script
 
 on:
   push:
-    branches: [ disabled ]
+    branches: [ actions ]
```
Comment (h2grover) [inline .github/workflows/amazon.yml]: @anjaliahlawat please revert it to `disabled`

File: .github/workflows/amazon.yml [hunk 2]
```diff
@@ -21,10 +21,12 @@ jobs:
...
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
-      run: mv .env.sample .env
       with:
-        node-version: ${{ matrix.node-version }}
+        node-version: ${{ matrix.node-version }}   
+    - name: Rename .env.sample file
+      run: mv .env.sample .env  
+    - name: Install dependencies
+      run: yarn
```
Comment (h2grover) [inline .github/workflows/amazon.yml]: @anjaliahlawat please rename the `.env` file and install dependencies under one stage name, let's say `Setup`

and another stage name will be `Running At's` for `yarn test`

**Review by h2grover — CHANGES_REQUESTED** [review]

**Review by h2grover — APPROVED** [review]
LGTM 👍 , Can you please share the job URL of the GitHub actions, then I will merge it.

**Comment by anjaliahlawat** [comment]
@h2grover https://github.com/anjaliahlawat/amazon-login-script/runs/2164890919?check_suite_focus=true
Again, one test passed and one failed.

**Pull request merged**